src/oversampling_methods/borderline_smote1/interface_mlj.jl

```diff
@@ -65,9 +65,20 @@ MMI.metadata_pkg(
 
 MMI.metadata_model(
     BorderlineSMOTE1,
-    input_scitype = Union{Table(Continuous),AbstractMatrix{Continuous}},
-    output_scitype = Union{Table(Continuous),AbstractMatrix{Continuous}},
-    target_scitype = AbstractVector,
+    input_scitype = Tuple{
+                        Union{
+                            Table(Continuous),
+                            AbstractMatrix{Continuous}
+                        }, 
+                        AbstractVector
+                    },
+    output_scitype = Tuple{
+        Union{
+            Table(Continuous),
+            AbstractMatrix{Continuous}
+        }, 
+        AbstractVector
+    },
```

Comment:
Whitespace suggestion:

```suggestion
    input_scitype = Tuple{
         Union{
            Table(Continuous),
            AbstractMatrix{Continuous}
        }, 
        AbstractVector
    },
    output_scitype = Tuple{
        Union{
            Table(Continuous),
            AbstractMatrix{Continuous}
        }, 
        AbstractVector
    },
```